rasa/server.py

```diff
@@ -137,6 +137,25 @@ def decorated(*args, **kwargs):
     return decorator
 
 
+def ensure_conversation_exists(app: Sanic):
+    """Wraps a request handler ensuring the conversation exists.
+    """
+
+    def decorator(f):
+        @wraps(f)
+        def decorated(*args, **kwargs):
+            conversation_id = kwargs["conversation_id"]
+            tracker = app.agent.create_processor().get_tracker(conversation_id)
+            if tracker is not None:
+                return f(*args, **kwargs)
+            else:
+                raise ErrorResponse(404, "Not found", "Conversation ID not found.")
```

Comment:
let's use a constant for this
```suggestion
                raise ErrorResponse(HTTPStatus.NOT_FOUND, "Not found", "Conversation ID not found.")
```